source/pe/3.8/r10k_reference.md

```diff
@@ -48,4 +48,4 @@ The `r10k puppetfile` command accepts the following subcommands:
 
 * `check`: Verifies the Puppetfile syntax is correct.
 * `install`: Installs all modules from a Puppetfile.
-* `purge`: Purges unmanaged modules from a Puppetfile managed directory.
```

Comment:
So the weird thing is that it wasn't.  Something odd appears to have happened in the GitHub interface.